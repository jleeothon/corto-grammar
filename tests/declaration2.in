list{int8} a
