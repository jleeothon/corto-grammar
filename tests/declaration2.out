(program (statement (declaration (typeLabel (anonymousType (identifier list) (initializerBraces { (initializerList (initializerElem (initializerValue (identifier int8)))) }))) a) \n))
